clusters/app.ci/openshift-user-workload-monitoring/mixins/_prometheus/prow_alerts.libsonnet

```diff
@@ -6,7 +6,7 @@
         rules: [
           {
             alert: 'prow-pod-crashlooping',
-            expr: 'increase(kube_pod_container_status_restarts_total{namespace="ci",job="kube-state-metrics"}[5m]) > 0',
+            expr: 'increase(kube_pod_container_status_restarts_total{namespace="ci",job="kube-state-metrics"}[5m]) > 0 and sum_over_time(kube_pod_container_status_restarts_total{namespace="ci",job="kube-state-metrics"}[5m]) > 5',
```

Comment:
We should also check why pod-scaler restarted so often. We should collect more solid proof that shows the alerts were noise. The alerts may indicate real issues.

With that said, we could also try.

```suggestion
            expr: 'increase(kube_pod_container_status_restarts_total{job="kube-state-metrics",namespace="ci"}[1h]) > 10',
```

The query comes from https://songrgg.github.io/operation/how-to-alert-for-Pod-Restart-OOMKilled-in-Kubernetes/ with our threshold `10`: a pod restarts 10 times in the last 1h.

I cannot justify 10. That is why I introduced the original query. As I said many times, reducing noise is easy and can be achieved by removing the alert. The hard part is avoiding overkills.

Results of the new query:

<img width="1523" alt="Screenshot 2023-01-04 at 9 21 38 AM" src="https://user-images.githubusercontent.com/4013349/210575500-3fdb3d89-ead3-446b-baa4-950a030628a1.png">

We should be good with the `for` expression in the alert.

If we hit overkills, we could tune the threshold again.